frontend/src/components/InstallModal.js

```diff
@@ -29,10 +29,19 @@ class InstallModal extends React.Component {
   };
 
   componentDidUpdate(prevProps) {
-    const { operator } = this.props;
+    const { operator, history } = this.props;
+    const path = history.location.pathname.split('/');
+
+    let channelName;
+    const operatorId = _.get(operator, 'id');
+
+    if (path.length === 4) {
+      [channelName] = path.slice(2);
```

Comment:
```suggestion
      channelName = path.slice(2);
```

Reply:
Path is the array and channelName is a string so we need to destructure array to obtain string.
Could be done using simple assignment (channelName = path[2]) but the way Linter is set does not allow it and force destructuring...